include/glow/Importer/ONNXModelLoader.h

```diff
@@ -123,6 +123,10 @@ class ONNXModelLoader
   /// A set of inputs which will be static placeholders.
   std::unordered_set<std::string> staticInputs_;
 
+  /// Load ONNX NonZero Operator.
```

Comment:
Can you add a disclaimer here explaining that Glow's requirement for static shapes means we require Constant input and do constant folding inline here in the loader?